app/javascript/components/shared/helpers.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const preserveLines = string => `${string}`.split("\n").map((line, idx) => <p key={idx}>{line}</p>);
+export const unnullifyFields = (object) => {
+  const result = Object.create(null);
```

Comment:
why not just simple `const result = {}`?

Reply:
`result` is always used as map, default prototype is unneeded.

Reply:
Yeah but it complicates the code, let's use the simple {} as prototype inclusion won't be a problem in here.